app/commands/vkick.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Execute on vkick command
+ * @param {Discord.Message} msg - discord message
+ * @param {Array} args - passed arguments
+ */
+const vkick = async (msg, args) => {
+    // get the user.
+    const user = msg.mentions.members.first();
+    // if user is not exist stop and display a message.
+    if (!user) {
+        return msg.channel.send(
+            `❌ Sorry ${msg.author.toString()}, We Couldn't find that member.`
+        );
+    }
+
+    if (!user.voice.channel) {
+        return msg.channel.send(
+            `❌ ${msg.author.toString()} Given user not in any voice channel.`
+        );
+    }
+    // voice unmute
+    if (user.voice.kick()) {
```

Comment:
After kicking the bot, the bot couldn't play the song then the bot server crashed.
```
vmute
[ '<@!747075798082650164>' ]
vkick
[
  'https://www.youtube.com/watch?v=XXYlFuWEuKI&ab_channel=TheWeekndVEVO'
]
play
[]
stop
TypeError: Cannot read property 'end' of null
TypeError: Cannot read property 'end' of null

/home/alotaima/Projects/side/SudoBot/node_modules/opusscript/build/opusscript_native_wasm.js:8
var Module=typeof Module!=="undefined"?Module:{};var moduleOverrides={};var key;for(key in Module){if(Module.hasOwnProperty(key)){moduleOverrides[key]=Module[key]}}Module["arguments"]=[];Module["thisProgram"]="./this.program";Module["quit"]=function(status,toThrow){throw toThrow};Module["preRun"]=[];Module["postRun"]=[];var ENVIRONMENT_IS_WEB=false;var ENVIRONMENT_IS_WORKER=false;var ENVIRONMENT_IS_NODE=false;var ENVIRONMENT_HAS_NODE=false;var ENVIRONMENT_IS_SHELL=false;ENVIRONMENT_IS_WEB=typeof window==="object";ENVIRONMENT_IS_WORKER=typeof importScripts==="function";ENVIRONMENT_HAS_NODE=typeof process==="object"&&typeof process.versions==="object"&&typeof process.versions.node==="string";ENVIRONMENT_IS_NODE=ENVIRONMENT_HAS_NODE&&!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS_WORKER;ENVIRONMENT_IS_SHELL=!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS_NODE&&!ENVIRONMENT_IS_WORKER;var scriptDirectory="";function locateFile(path){if(Mod
abort(TypeError: Cannot read property 'end' of null). Build with -s ASSERTIONS=1 for more info.
```